### DISCUSSION TITLE
请问用 s 能部署 NAS 吗？

### DISCUSSION THREAD

**Discussion by g8up** [Serverless-Devs/Serverless-Devs#437] [General]
服务器里需要一个字体文件（ttf），想通过 NAS 上传后然后引入到系统。

**Comment by heimanba** [comment]
可以的，可以看下s nas这个指令，或者到serverless-devs.com看下使用文档哈



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年3月21日(周一) 晚上9:18
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: [Serverless-Devs/Serverless-Devs] 请问用 s 能部署 NAS 吗？ (Discussion #437)




 
服务器里需要一个字体文件（ttf），想通过 NAS 上传后然后引入到系统。

**Reply by heimanba** [reply]
https://www.serverless-devs.com/fc/command/nas

**Comment by anycodes** [comment]
可以把字体放在nas，然后在程序中引用，可以通过s nas upload命令使用
